src/js/jsx/sections/transform/Rotate.jsx

```diff
@@ -50,18 +52,36 @@ define(function (require, exports, module) {
          */
         _lastAngle: null,
```

Comment:
This seems like it has been replaced by `state.value`.  It seems we are maintaining it for no reason now?